### PR TITLE
Remove rspec and add minitest

### PR DESCRIPTION
**Pull request by abhaynikam**
Related to: https://github.com/rubocop-hq/rubocop-minitest/issues/1#issuecomment-518548748

PR fixes: 
- Removes rspec and initial specs added
- Setups minitest and migrates the specs to minitest.

Screenshot: 
rubocop-minitest

### CODE REVIEW

**Comment by koic** [comment]
We can remove .rspec file.
https://github.com/rubocop-hq/rubocop-minitest/blob/master/.rspec

**Comment by abhaynikam** [comment]
@koic : Done with the changes.

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,8 @@
-require "bundler/gem_tasks"
```
Comment (koic) [inline Rakefile]: It doesn't depend on RSpec, so it doesn't seem to need to be removed.

File: rubocop-minitest.gemspec [hunk 2]
```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 2.0"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "minitest", "~> 5.11"
```
Comment (koic) [inline rubocop-minitest.gemspec]: The latest version of minitest is specified.
https://rubygems.org/gems/minitest/versions
Is there any reason not to support older versions?

Comment (abhaynikam) [reply rubocop-minitest.gemspec]: No reason. I just picked it from the rubygems. Can you let me know which version we should go with? I can see Rails 4.0 has minitest `4.7.5`

Comment (koic) [reply rubocop-minitest.gemspec]: Thanks! Let's see how it goes :-)

**Comment by abhaynikam** [comment]
Thanks 🎉

**Pull request merged**